README.md

```diff
@@ -1,6 +1,106 @@
+    - [CheckRebootHygiene](#checkreboothygiene)
+  - [Building the Custom Actors RPM](#building-the-custom-actors-rpm)
+    - [Prerequisites](#prerequisites)
+    - [Building the RPM](#building-the-rpm)
+    - [Installing the RPM](#installing-the-rpm)
+    - [Using the Custom Actors](#using-the-custom-actors)
+  - [Contributing](#contributing)
+
+## Actor list
+
+The following actors are currently available in this repository:
+- checkreboothygiene
+
+These actors are listed in [`actor-list.txt`](./actor-list.txt) file by default.
+
+## Actor description
+
+### CheckRebootHygiene
+
+The CheckRebootHygiene actor will report an inhibitor in case of conditions
+that may be prohibited by an organization's reboot hygiene policy.
+
+The actor will report inhibitor risk when:
+* The host uptime is greater than the maximum defined by the policy.
+* The running kernel version does not match the default kernel version configured in the bootloader.
+* Any files are found under /boot that have been modified since the last reboot.
+
+
+
+## Building the Custom Actors RPM
+
+In order to bundle the custom actors from this Git repository, follow the instructions below.
+
+### Prerequisites
+
+- A system running Red Hat Enterprise Linux 7 or 8
+- `git` installed on your system
+- `make` package installed
+- `rpm-build` package installed
+- Appropriate Python development package
+
+You can install these prerequisites by running the following command:
+
+- On RHEL 7:
+
+    ```bash
+    sudo yum install git make rpm-build python2-devel
+    ```
+
+- On RHEL 8:
+
+    ```bash
+    sudo dnf install git make rpm-build python3-devel
+    ```
+
+### Building the RPM
+
+1. Clone the repository to your local machine:
+```bash
+git clone https://github.com/oamg/leapp-supplements.git
+```
+
+2. Change to the `leapp-supplements` directory:
+```bash
+cd leapp-supplements
+```
+
+3. Ensure the `actor-list.txt` file contains the actors you want to bundle. Each actor should be listed on a separate line. Lines starting with `#` are treated as comments and will be ignored. Feel free to add or remove actors as needed for your specific needs.
+
+4. Build the RPM package by running the following command:
+```bash
+make rpmbuild
+```
+
+After the process is complete, you should see the generated RPM files in the current directory.
+
+### Installing the RPM
+
+To install the custom actors RPM, run the following command:
+```bash
+sudo yum install ./<generated_rpm_file>
+```
+
+Replace `<generated_rpm_file>` with the actual RPM file generated in the previous step.
+
+### Using the Custom Actors
+
+Once the RPM is installed, the custom actors are automatically integrated with the Leapp project. You can now perform in-place upgrades using the Leapp command-line interface. The custom actors from this repository will be used during the upgrade process. For more details, they can be seen installed under `/usr/share/leapp-repository/repositories/system_upgrade_supplements`.
```

Comment:
In this case should be used `/usr/share/leapp-repository/custom-repositories/system_upgrade_supplements`  as mentioned here: https://access.redhat.com/articles/4977891

Reply:
I was thinking wether to put it in the repositories or custom-repositories since it could clash with other custom repos of customers (not coming from this repo, not sure how often that is the case... - probably low), but coming back to this I think it should be in custom-repositories as you say. There is a low probability of this clash since there would be another repository `system_upgrade_supplements` under `custom-repositories` so there is definitely room for more folders and actors...

will change